highfidelity/hifi
# Photobooth Improvements - Stage, Camera Angles, and Lights

To Test: 
- follow the same instruction from https://github.com/highfidelity/hifi/pull/9174
- make sure your avatar stands inside of the zone so that lighting works correctly. If you stand on the stage(white big platform), it should be good.
- confirm that the framing now looks better
